%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Zach Schreiber Viral dark matter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\documentclass{beamer}

\mode<presentation>
{
  \usetheme{Madrid}   
  \usecolortheme{default}
  \usefonttheme{serif}    
  \setbeamertemplate{navigation symbols}{}
  \setbeamertemplate{caption}[numbered]
} 

\usepackage[english]{babel}
\usepackage[utf8x]{inputenc}
\usepackage{chemfig}
\usepackage[version=3]{mhchem}
\usepackage{mathtools}
\usepackage{graphicx,caption}
%comment them out before you export
\usepackage{pgfpages}
\pgfpagesuselayout{resize to}[%
  physical paper width=8in, physical paper height=6in]

% Here's where the presentation starts, with the info for the title slide
\title[]{Viral dark matter project}
\author[]{Zach Schreiber}
\institute[]{Delaware Biotechnology Institute}
\date{\today}

\begin{document}


\section{Introduction}

\begin{frame}{Introduction}

\begin{itemize}
  \item There are around 10\textsuperscript{31} viruses infecting host populations.
  \item 15:1 ratio between aquatic phage and bacterial host.
  \item As of 2015, under 2,220 dsDNA and retroviral genomes are available in the NCBI database vs 45,000       		        bacterial genomes.
  	\begin{itemize}
  		\item 85 \% of sampled phage are dsDNA targeted
  	\end{itemize}
  
\end{itemize}

\begin{figure}
\centering
\includegraphics[height=3.6cm, width=6cm]{images/a.png}\\[-1ex]
{\tiny (Paez-Espino et al., 2016)}
\caption{Environmental distribution of metagenomic viruses}
\label{fig:1}
\end{figure}

\end{frame}

\section{Importance}

\begin{frame}{Why study viruses?}

Pros and cons of viral metagenomic research:
\begin{block}{\texttt{Pros}}
\begin{itemize}
  \item Act as ecological drivers within their host populations
  	\begin{itemize}
  		\item Horizontal gene transfer, metabolism, antibiotic resistance
  	\end{itemize}
  \item Advancing technology
  \item Still vastly unexplored 
\end{itemize}

\end{block}
\begin{block}{\texttt{Cons}}

\begin{itemize}
  \item Possibly no universal marker genes unlike bacteria
  \item Lack of reference databases (changing)
  \item Hard to taxonomically classify due to abundance and rapid turnover times
  \item File sizes can limit what type of analysis you can perform
\end{itemize}

\end{block}

\end{frame}

\begin{frame}{Why study viruses?}

\begin{figure}
\centering
\includegraphics[height=5cm, width=11cm]{images/4.png}\\[-1ex]
{\tiny (Hurwitz and Sullivan, 2013)}
\caption{Taxonomic distribution of POV reads}
\label{fig:2}
\end{figure}

\end{frame}

\section{Sequencing}


\begin{frame}[fragile]
\frametitle{Sequencing methods and tools used in viral metagenomics}

\begin{itemize}
\item \texttt{Illumina}, \texttt{PacBio}, \texttt{Sanger}, \texttt{454}, \texttt{Ion Torrent}, \texttt{Genia}
\item Each have their advantages and disadvantages, however, \texttt{Illumina} and \texttt{PacBio} seem to be the          	  preferred for high throughput data.  
\end{itemize}


\begin{block}{\texttt{Available tools}}

\begin{itemize}
  \item VIROME, MgOl, MG-RAST, MEGAN, METAVIR etc.. 
  \item Each have their own specific uses
  \item Still do not address this issue of viral dark matter (unassigned)
\end{itemize}
\end{block}
\end{frame}

\section{Viraldm}

\begin{frame}[fragile]
\frametitle{Viral dark matter introduction}
\texttt{Viruses are most abundant biological entity on Earth...}

\begin{itemize}
\item Difficult to classify taxonomically, isolate, sampling.  
\item Limited reference databases, high diversity, lack of genetic markers. 
\end{itemize}

\texttt{Viral dark matter classified as...}
\begin{itemize}
\item Unannotated functional and or structural viral genes.
\item 63 to 93\% of surveyed sequence.
\item Mainly dsDNA targeted.
\end{itemize}


\end{frame}

\begin{frame}[fragile]
\frametitle{Viral dark matter project}
\texttt{Project objectives: }

\begin{itemize}
\item Better understand viral dark matter through network-based and clustering analytics.
\item Large scale datasets
\item Possibly develop a web-application tool for VIROME
\end{itemize}


\begin{figure}
\centering
\includegraphics[height=3.6cm, width=11cm]{images/network1.png}\\[-1ex]
\caption{Cytoscape network of clustered mmi libraries}
\label{fig:3}
\end{figure}

\end{frame}

\section{virome}
\begin{frame}[fragile]
\frametitle{VIROME}
\texttt{Viral Informatics Resource for Metagenome Exploration}

\begin{itemize}
\item Pipeline developed to analyze predicted ORF's in viral metagenomes with emphasis on environmental metadata.
\item Quality control sequence and assign gene function and taxonomy after a BLAST-P against UniRef100 and MgOl.
\item Display results to user on the VIROME web application.

\begin{figure}
\centering
\includegraphics[height=3.6cm, width=11cm]{images/virome.png}\\[-1ex]
\caption{VIROME ORF and sequence categories}
\label{fig:3}
\end{figure}

\end{itemize}

\end{frame}


\section{workflow}
\begin{frame}[fragile]
\frametitle{Work-flow}

\begin{itemize}
\item Step 1: Cluster incoming sequences using UniRef and MgOl reference databases.

	 \item Cd-hit clusters proteins based off sequence identity specified by user. 
	 	 \begin{itemize}
	 \item Short word filtering algorithm which determines similarity  between two sequences without performing an alignment (Very fast for metagenomic data). 
	 \item Each cluster has at least one representative sequence 
	 \item Outputs a .cluster file containing list of clusters
	 \end{itemize}
\item Step 2: Parse through list of clusters and generate a file that will be used in cytoscape (Based off header format).
\item Step 3: Build and display network tree.

\end{itemize}

\end{frame}


\section{data}
\begin{frame}[fragile]
\frametitle{Types of input and output}
\includegraphics[height=3.2cm, width=12cm]{images/output.png} 
\begin{itemize}
\item Box 1: Cd-hit output file representing cluster number along with header id and \% matched.
\item Box 2: Cytoscape input file showing how often two clusters shared the same header id.
\item Box 3: View of network.
\end{itemize}

\end{frame}


\section{Future work}

\begin{frame}[fragile]
\frametitle{Future Work}


\begin{itemize}
\item Tool that would display a network tree based off user input (Utilizing cytoscape.js)
\item The user would be able to filter network based off various metadata values selected.
\item Binning data from network tree values (New database). 

\end{itemize}

\texttt{Other projects}
\begin{itemize}
\item Addition of savior onto farber.hpc.udel.edu.
\item Migration of VIROME and MgOl to Laravel framework. 
\end{itemize}

\end{frame}


\end{document}